### ISSUE TITLE
[Priority 2/2] SM3.95-compatible Judgment Tracking

### ISSUE THREAD

**Issue by DivineJayce** [DivinElegy/GrooveNights#35]
This mostly relates to working out when a player gets their first excellent, great and miss so it can be displayed in the graph on the results screen.

An accurate tracking process needs to be identified before a few other desired systems can be implemented.

===Old Stuff that was discussed but isn't all that relevant now===

> > We need to implement all the Judgment Things found in Mods.lua of SIMPLY LOVE since the
> > STATSMAN:GetCurStageStats():GetPlayerStageStats(0):GetTapNoteScores(TNS_GOOD);
> > stuff is OpenITG only.
> > 
> > So basically JudgmentInit(), JudgmentTween(), JudgmentCommand(), TrackJudgment(), TrackHold(), CheckHold(), AssignHold(), JudgmentCapture(), CaptureMeter() and anything else that's relevant.
> > 
> > Following on from that it doesn't look like Simply Love implemented a way to track if a player hits a mine. The easiest way I can think to do it would be to see if the player's Life decreases (but their Miss+WayOff+Hold count stays the same), since the only other thing that could drop the life bar is a mine in that case.

**Comment by ghost** [comment]
Maybe I'm retarded but I can't see how Simply Love implemented any of this. For example, the function TrackJudgment is only called from the function JudgmentCommand, and the function JudgmentCommand is never called as far as I can tell... Am I missing something?

**Comment by DivineJayce** [comment]
Yeah they never had a need to call it, as far as I can tell they just implemented the functionality in case they ever wanted to call it.

I have no idea whether it works or not, mind you
